integration-tests/cucumber-tests-platform-distributionautomation/src/test/java/org/opensmartgridplatform/cucumber/platform/distributionautomation/PlatformDistributionAutomationKeys.java

```diff
@@ -11,28 +11,30 @@
 
 public class PlatformDistributionAutomationKeys {
 
+    public static final String BAY_IDENTIFICATION = "bay identification";
```

Comment:
In general it is probably inadvisable to even have files like these? 
I thought it would be clearer to have them ordered alphabetically. In my opinion it makes it easier to see whether a constant (or a constant similar to a possible new constant) is already present in the file.
Logical grouping could also make sense, but what if a certain constant could be part of multiple logical groups?

Reply:
In this particular file I don't see more logical grouping or sorting being broken, so I don't mind, but I would advice not to make it a standard reflex to order constants alphabetically since I know of some other files where such logical groupings do exist. I think it makes more sense to apply informed judgement than to go for a convention of alphabetical ordering.